crates/ika-core/src/sui_connector/sui_event_into_request.rs

```diff
@@ -208,16 +210,43 @@ fn dwallet_dkg_session_request(
     deserialized_event: DWalletSessionEvent<DWalletDKGRequestEvent>,
     pulled: bool,
 ) -> DwalletMPCResult<DWalletSessionRequest> {
-    Ok(DWalletSessionRequest {
-        session_type: deserialized_event.session_type,
-        session_identifier: deserialized_event.session_identifier_digest(),
-        session_sequence_number: deserialized_event.session_sequence_number,
-        protocol_data: dwallet_dkg_protocol_data(deserialized_event.event_data.clone())?,
-        epoch: deserialized_event.epoch,
-        requires_network_key_data: true,
-        requires_next_active_committee: false,
-        pulled,
-    })
+    return match &deserialized_event.event_data.user_secret_key_share {
```

Comment:
better leave the `user_secret_key_share` as-is, or, convert it to the BytesUserSecretKeyShareVerification here, this way don't need two options